otter/test/convergence/test_steps.py

```diff
@@ -469,9 +469,9 @@ def test_add_nodes_to_clb_terminal_failures(self):
         if there is any other 4xx error, then
         the error is propagated up and the result is a failure.
         """
-        terminals = (CLBDeletedError(lb_id=u"12345"),
+        terminals = (CLBNotFoundError(lb_id=u"12345"),
+                     CLBDeletedError(lb_id=u"12345"),
                      CLBNodeLimitError(lb_id=u"12345"),
-                     NoSuchCLBError(lb_id=u"12345"),
```

Comment:
Sorry, I meant in the test code just to be sure that both the errors are handled. I can understand if you consider that duplicate effort knowing the fact the `CLBNotFoundError` is based and it is checked. However, I am just thinking of the test as black box not knowing about the inheritance which is why I suggested no changes in them in the first place. This is just bike-shedding. I am actually fine either way :). This LGTM if you don't change it.